### PR TITLE
Fix Windows test errors in src/ls.js and test/cp.js.

### PR DESCRIPTION
**Pull request by kmashint**
These are fixes for Windows in the switch-fast-blob branch as mentioned in this pull-request.
https://github.com/shelljs/shelljs/pull/1153

### CODE REVIEW

**Review by nfischer — APPROVED** [review]

**Comment by nfischer** [comment]
@kmashint can you please rebase this PR on top of the `master` branch instead? I realized this change is missing some critical commits that I landed recently on `master`. I'm hoping that your fixes are backward compatible with both `glob` and `fast-glob` so this should be relatively easy to land.

**Comment by kmashint** [comment]
@nfischer Will rebase on master tonight.

**Comment by kmashint** [comment]
I merged master into this pull-request branch, and hoping I cut it the right way, removing references to glob in favour of fast-glob.

File: src/common.js [hunk 1]
```diff
@@ -246,12 +246,6 @@ function parseOptions(opt, map, errorOptions) {
 }
 exports.parseOptions = parseOptions;
 
-function globOptions() {
```
Comment (nfischer) [inline src/common.js:250]: I think this was reverted when it should not have been. I think what happened is that you probably have an out-of-date copy of the `master` branch. Can you please sync this against the main repo's master branch? I think you can do this with something like:

```
git checkout master
git remote add upstream https://github.com/shelljs/shelljs.git
git fetch upstream
git rebase upstream/master
git checkout switch-fast-glob
git rebase master
git push origin switch-fast-glob --force-with-lease
```

**Review by nfischer — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -55,7 +55,7 @@
   },
   "dependencies": {
     "execa": "^1.0.0",
-    "glob": "^7.0.0",
+    "fast-glob": "^3.3.2",
```
Comment (nfischer) [inline package.json:58]: Also, I would like to exclude this part from this PR. My change will take care of switching over to `fast-glob`. I'd like to merge your change before mine though to make sure it is backward-compatible with "plain" `glob. Then I'll rebase my change on top of yours to test it with `fast-glob`.

It's also fine if you want to test this with `fast-glob` locally in your checkout. We should just remove this part before merging this on the `master` branch.

**Comment by nfischer** [comment]
Closing this one in favor of #1166.

**Pull request closed without merging**